src/main/java/org/folio/rest/impl/DataExportImplFileDefinitionImpl.java

```diff
@@ -107,4 +110,11 @@ public void getDataExportFileDefinitionsByFileDefinitionId(String fileDefinition
       .setHandler(asyncResultHandler);
 
   }
+
+  private Future<Void> validateFileNameExtension(String fileName) {
+    if (!FilenameUtils.isExtension(fileName.toLowerCase(), CSV_FORMAT_EXTENSION)) {
+      throw new ServiceException(HttpStatus.HTTP_UNPROCESSABLE_ENTITY, "File name extension does not corresponds csv format");
```

Comment:
Can you add this to the ErrorCode enum class, so that we have all error messages in a single place

Reply:
done